en/05_Contributing/06_Documentation.md

```diff
@@ -1,141 +1,250 @@
 ---
-title: Documentation
-summary: Writing guide for contributing to SilverStripe developer and CMS user help documentation. 
+title: Contributing Documentation
+summary: Guide for contributing to Silverstripe CMS developer and CMS user help documentation.
 icon: file-alt
 ---
 
 # Contributing documentation
 
-Documentation for a software project is a continuing, collaborative effort. We encourage everybody to contribute in any way they can, from simply fixing spelling mistakes, to writing recipes, reviewing existing documentation and translation to other languages.
+Documentation for a software project is a continuous, collaborative effort. We encourage everybody to contribute in any way they can whether it's fixing a typo, updating some outdated documentation, or adding missing documentation.
 
-Modifying documentation requires basic knowledge of [PHPDoc](http://en.wikipedia.org/wiki/PHPDoc) and 
-[Markdown](http://daringfireball.net/projects/markdown/) as well as a GitHub user account.
+Modifying documentation requires basic knowledge of [markdown](https://www.markdownguide.org/). Our documentation also includes some additional syntax which you can read about in the [extended markdown syntax](#extended-markdown-syntax) section below.
```

Comment:
- I'm removing phpdocs from this page - that belongs in the contributing code page.
- The old markdown link doesn't show any of the extended syntax like fenced code blocks or tables which we take advantage of.